Vagrantfile

```diff
@@ -104,15 +104,40 @@ Vagrant.configure("2") do |config|
 
   SHELL
 
+  ######################################################################
+  # Setup a Bluemix and Kubernetes environment
+  ######################################################################
+  config.vm.provision "shell", inline: <<-SHELL
+    echo "\n************************************"
+    echo " Installing IBM Cloud CLI..."
+    echo "************************************\n"
+    # Install IBM Cloud CLI as Vagrant user
+    # curl -fsSL https://clis.cloud.ibm.com/install/linux | sh
+    sudo -H -u vagrant sh -c 'curl -sL http://ibm.biz/idt-installer | bash'
+    sudo -H -u vagrant sh -c 'ibmcloud config --usage-stats-collect false'
+    sudo -H -u vagrant sh -c "echo 'source <(kubectl completion bash)' >> ~/.bashrc"
+    sudo -H -u vagrant sh -c "echo alias ic=/usr/local/bin/ibmcloud >> ~/.bash_aliases"
+    echo "\n"
+    echo "If you have an IBM Cloud API key in ~/.bluemix/apiKey.json"
+    echo "You can login with the following command:"
+    echo "\n"
+    echo "ibmcloud login -a https://cloud.ibm.com --apikey @~/.bluemix/apiKey.json -r us-south"
+    echo "\n"
+    echo "\n************************************"
+    echo " For the Kubernetes Dashboard use:"
+    echo " kubectl proxy --address='0.0.0.0'"
+    echo "************************************\n"
+  SHELL
+  
   # Add MongoDB docker container
   # docker run --name some-mongo -v /my/own/datadir:/data/db -d mongo
   # To use interactive bash: docker exec -it mongodb bash
   # The MongoDB Server log is available: docker logs mongodb
-  #config.vm.provision "docker" do |d|
-  #  d.pull_images "mongo"
-  #  d.run "mongo",
-  #    args: "--name mongodb -d -p 27017:27017 -v mongo_data:/data/db "
-  #end
+  config.vm.provision "docker" do |d|
```

Comment:
Since we have migrated to the cloud database so the docker container for the local db seems not necessary?